CONTRIBUTING.md

```diff
@@ -95,28 +95,64 @@ yarn dev --force
 
 > You can now hack in the cloned repository of `api-platform-admin`.
 
-#### Running Admin through Storybook
+#### Running Admin Through Storybook
 
-If you don't have an existing API Platform application, or don't want to use `yarn link`, you can use one of the bundled example APIs, and visualize the admin through [Storybook](https://storybook.js.org/).
+If you do not have an existing project, you can contribute by visualizing directly the changes you made the sources through [Storybook](https://storybook.js.org/). 
+
+This development stack consists of two Docker containers: 
+- `pwa`: containing the `<Admin>` sources and Storybook;
+- `php`: holding the API sources.
+
+Additionally, this method allows testing the integration between Api-platform and the `admin` component by writing stories, scenarios and tests.
 
 Install everything:
 
 ```shell
 docker compose up
 ```
 
-Go to http://localhost, accept the self-signed certificate, and see the running Admin.
+Before accessing the Storybook instance, make sure to go to https://localhost to accept the self-signed certificate. Once it's done, you'll see the API documentation running on a customized version of Swagger UI.
 
-To stop and prune the simple API Platform backend:
+Now you can go to http://localhost:3000/ to see the Storybook instance in action. The changes you'll make in the source code will be hot-reloaded.
+
+To access run a command directly inside a container, run:
+```shell
+# Run a command in the php container 
+docker compose exec -T php your-command
+
+# Run a command in the pwa container
+docker compose exec -T pwa your-command
+```
 
 ### Testing Your Changes
 
-Before sending a Pull Request, make sure the tests pass correctly:
+Before sending a Pull Request, make sure the tests pass correctly.
+
+#### With Linked Sources
+
+If you have chosen to link `admin` sources, simply run:
 
 ```console
 yarn test
 ```
 
+#### With Admin Through Storybook
+
+If you have chosen to develop with Storybook, you have to build the CI stack first:
+
+```console
+docker compose docker compose -f compose.yaml -f compose.ci.yaml up
+```
+
+Once the containes are healthy, run 
+```console
+# fonctional tests
+docker compose exec -T pwa yarn test
+
+# End to end tests (Running with Storybook interactions https://storybook.js.org/docs/writing-stories/play-function)
+docker compose exec -T pwa yarn storybook:test --url http://127.0.0.1:3000
```

Comment:
I realize that's because I assumed you made a copy paste error here: https://github.com/api-platform/admin/pull/541#discussion_r1557841834
But actually using the ci image was intended, and with that config it works. 
Still, I find it kind of odd to use the ci config even for local development. Maybe we should organize our docker config differently?

Reply:
Indeed, as Docker is only used for dev, maybe can we merge `compose.yaml` and `compose.ci.yaml`. This will simplify everything.